src/102610_len_2330.c

```diff
@@ -61,20 +60,16 @@ void func_802E1204(Entity* entity) {
 }
 
 void func_802E1270(Entity* entity) {
-    s32* temp = &D_802EB390;
-
-    if (*temp != 0) {
+    if (D_802EB390 != 0) {
         exec_entity_updatecmd(entity);
     }
 }
 
 void func_802E1298(void) {
-    s32* temp1 = &D_802EB390;
-    s32* temp2 = &D_802EB394;
-    *temp1 = 0;
-    *temp2 = 0;
-    D_802EB39C = load_string(0x1D0004, temp2);
-    D_802EB398 = load_string(0x1E000A, temp1);
+    D_802EB390 = 0;
+    D_802EB394 = 0;
+    D_802EB39C = load_string(0x1D0004, &D_802EB394);
+    D_802EB398 = load_string(0x1E000A, &D_802EB390);
```

Comment:
We should parameterise `0x1D0000`, `0x1D0004`, and `0x1E000A` via `message_ids.h` (see also its configuration, `tools/splat_ext/msg.yaml`). I don't mind if you don't do that in this PR, but it'd be preferable 😛

Here are the messages (which also give nice names for the unnamed `D_` variables and the functions themselves):
```
#message:1D:000 {
	[Style inspect]
	Hit this block to save your[BR]
	game progress.[BR]
	[Wait][Next]
	When the game is over, your[BR]
	status will return to what it[BR]
	was the last time you saved,[BR]
	and you can continue to play.[BR]
	[Wait][Next]
	When you find this block along [BR]
	the way, remember to hit it[BR]
	to save![Wait][End]
}

#message:1D:004 {
	[Style inspect]
	Do you want to save?[BR]
	[Yield][End]
}

#message:1E:00A {
	[Style choice pos=110,90 size=115,46]
	[DelayOff]
	[Cursor 0][Option 0]Save[BR]
	[Cursor 1][Option 1]Don't save[BR]
	[Option 255][DelayOn]
	[SetCancel 1][EndChoice 2][End]
}
```